build.yaml

```diff
@@ -3807,6 +3798,55 @@ steps:
       - genetics_public_image
     scopes:
       - dev
+  - kind: runImage
+    name: test_hail_fs
+    image:
+      valueFrom: hail_run_tests_image.image
+    resources:
+      memory: standard
+      cpu: '2'
+    script: |
+      set -ex
+      cd /io
+      tar xzf resources.tar.gz -C src/test
+
+      export CLOUD={{ global.cloud }}
+      export GOOGLE_APPLICATION_CREDENTIALS=/test-gsa-key/key.json
+      export HAIL_FS_TEST_CLOUD_RESOURCES_URI={{ global.test_storage_uri }}/{{ upload_test_resources_to_blob_storage.token }}/test/resources/fs
+      export HAIL_TEST_GCS_BUCKET={{ global.hail_test_gcs_bucket }}
```

Comment:
This currently doesn't exist in azure, but I can see that it's necessary for the FS tests. I'll lump it in with the test keys to get this into the infrastructure, but maybe we should mark this as gcp-only for now? Jinja will barf about this being undefined.

Reply:
Ah, hmm. I think Jinja will not complain if I hide it behind an if.